test/docker-compose.yml

```diff
@@ -27,7 +27,7 @@ services:
     <<: *app_default
     environment:
       <<: *base_env
-      S3_TARGET_BUCKET_URL: s3://app_restore/
+      TARGET_BUCKET_URL: s3://app_restore/
       S3_TARGET_FILE: dummy-backup-20180622.tar.bz2
```

Comment:
Please modify `S3_TARGET_FILE` to `TARGET_FILE`.

Reply:
Done.